doc/terminology.adoc

```diff
@@ -0,0 +1,9 @@
+= Terminology
+
+== Lua
+
+Class:: A metatable that sets `__index`
```

Comment:
Classes are rather an abstract construct. Metatables are used to implement classes, but not the classes themselves.
Maybe just link to wikipedia here.